code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1235,3 +1235,82 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_icon_state = "glass_yellow"
 	glass_name = "Eggnog"
 	glass_desc = "For enjoying the most wonderful time of the year."
+
+// IPC ALCHOHOL WOOOOOOOOOOO
+/datum/reagent/consumable/ethanol/synthanol
+	name = "Synthanol"
+	id = "synthanol"
+	description = "A runny liquid with conductive capacities. Its effects on synthetics are similar to those of alcohol on organics."
+	reagent_state = LIQUID
+	color = "#1BB1FF"
+	process_flags = ORGANIC | SYNTHETIC
```

Comment:
You let both organics and synthetics get drunk on synthanol by not changing the processing flags. That's not gonna fly.

Ideally, you should make it so that Synthanol only gets IPCs drunk, while inducing vomiting or something in humans. No clean way to do this while making Synthanol a child of regular alcohol. I'd copypaste the `on_mob_life` of regular ethanol and add some checks for robotic drinkers.

Reply:
1: I am not even sure if process_flags does anything on TG code
2: How?